src/Microsoft.AspNetCore.Razor.Language/Legacy/CSharpCodeParser.cs

```diff
@@ -21,17 +21,29 @@ internal class CSharpCodeParser : TokenizerBackedParser<CSharpTokenizer, CSharpS
         internal static readonly DirectiveDescriptor AddTagHelperDirectiveDescriptor = DirectiveDescriptor.CreateDirective(
             SyntaxConstants.CSharp.AddTagHelperKeyword,
             DirectiveKind.SingleLine,
-            builder => builder.AddStringToken());
+            builder =>
+            {
+                builder.AddStringToken();
+                builder.Description = "Add tag helper(s) to the current document.";
+            });
 
         internal static readonly DirectiveDescriptor RemoveTagHelperDirectiveDescriptor = DirectiveDescriptor.CreateDirective(
             SyntaxConstants.CSharp.RemoveTagHelperKeyword,
             DirectiveKind.SingleLine,
-            builder => builder.AddStringToken());
+            builder =>
+            {
+                builder.AddStringToken();
+                builder.Description = "Remove tag helper(s) to the current document.";
+            });
 
         internal static readonly DirectiveDescriptor TagHelperPrefixDirectiveDescriptor = DirectiveDescriptor.CreateDirective(
             SyntaxConstants.CSharp.TagHelperPrefixKeyword,
             DirectiveKind.SingleLine,
-            builder => builder.AddStringToken());
+            builder =>
+            {
+                builder.AddStringToken();
+                builder.Description = "Specify a prefix that is required for an element to be considered a tag helper.";
```

Comment:
"Specify a prefix that is required in an element name for it to be included in Tag Helper processing."